mmv1/products/firebase/terraform.yaml

```diff
@@ -57,6 +57,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           - project
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       custom_import: templates/terraform/custom_import/self_link_as_name.erb
+  AndroidApp: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ['{{project}} {{name}}']
+    autogen_async: true
+    skip_delete: true #currently only able to delete a androidapp through the Firebase Admin console
```

Comment:
as with AppleApp I think we should enable delete from the start.

Reply:
Done.